spring-cloud-dataflow-admin/src/test/java/org/springframework/cloud/dataflow/admin/AdminApplicationTests.java

```diff
@@ -64,5 +65,15 @@ public void testLatticeProfile() {
 		assertThat(context.getBean("latticeProperties", LatticeProperties.class).getReceptor().getHost(), is("receptor.52.7.176.225.xip.io"));
 		context.close();
 	}
+	
+	@Test
+	public void testKubernetesProfile() {
+		SpringApplication app = new SpringApplication(AdminApplication.class);
+		ConfigurableApplicationContext context = app.run(new String[] { "--spring.profiles.active=kubernetes",
+				"--server.port=0" });
```

Comment:
Can set port to -1 to even not bother creating a webserver (speeding up the test). May apply to other tests